hack/rebase-to-origin.sh

```diff
@@ -19,29 +31,27 @@ exclude_pkgs=(
   pkg/version
 )
 
-s2i_dir=$GOPATH/src/github.com/openshift/source-to-image
-origin_dir=$GOPATH/src/github.com/openshift/origin
-origin_s2i_godep_dir=${origin_dir}/Godeps/_workspace/src/github.com/openshift/source-to-image
-s2i_ref=$(cd ${s2i_dir} && git rev-parse --verify HEAD)
-s2i_short_ref=$(cd ${s2i_dir} && git rev-parse --short HEAD)
-s2i_godeps_ref=$(grep -m1 -A2 'openshift/source-to-image' ${origin_dir}/Godeps/Godeps.json | \
-  grep Rev | cut -d ':' -f2 | sed -e 's/"//g' | sed -e 's/^[[:space:]]*//')
+readonly origin_s2i_godep_dir="${OS_ROOT}/Godeps/_workspace/src/github.com/openshift/source-to-image"
+readonly s2i_ref="$(git -C ${S2I_ROOT} rev-parse --verify HEAD)"
+readonly s2i_short_ref="$(git -C ${S2I_ROOT} rev-parse --short HEAD)"
+readonly s2i_godeps_ref="$(grep -m1 -A2 'openshift/source-to-image' ${OS_ROOT}/Godeps/Godeps.json |
+  grep Rev | cut -d ':' -f2 | sed -e 's/"//g' -e 's/^[[:space:]]*//')"
 
-pushd "${origin_dir}" >/dev/null
-  git checkout -b "s2i-${s2i_short_ref}-bump"
+pushd "${OS_ROOT}" >/dev/null
+  git checkout -B "s2i-${s2i_short_ref}-bump" master
   rm -rf "${origin_s2i_godep_dir}/*"
-  cp -R ${s2i_dir}/pkg ${origin_s2i_godep_dir}/.
+  cp -R "${S2I_ROOT}/pkg" "${origin_s2i_godep_dir}/."
 
   # Remove all test files
-  find ${origin_s2i_godep_dir} -type f -name '*_test.go' -delete
+  find "${origin_s2i_godep_dir}" -type f -name '*_test.go' -delete
```

Comment:
let's hold on merging this until we find out if we should be removing the test files or not.